presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1532,6 +1536,32 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
     }
 
     @Override
+    public List<GrantInfo> listTablePrivileges(ConnectorSession session, String schemaNameOrNull, String grantee)
```

Comment:
Oh, I think my internet was flaky when leaving this comment: I think it would be better to use QualifiedTablePrefix here, to match with other information schema methods (e.g. the list* methods)